config/main.py

```diff
@@ -394,6 +394,18 @@ def _get_platform():
                 return tokens[1].strip()
     return ''
 
+def _get_sonic_generated_services():
+    if not os.path.isfile('/etc/sonic/generated_services.conf'):
+        return None
+    generated_services_list = []
+    with open('/etc/sonic/generated_services.conf') as generated_service_file:
+        for line in generated_service_file:
+            generated_services_list.append(line.rstrip('\n'))
+
+    return None if not generated_services_list else generated_services_list
+
+
+
```

Comment:
Delete one blank line (leave two blank lines between functions per pep8 standard)

Reply:
Thanks @jleveque . Will keep in mind from next time.